server/config.go

```diff
@@ -0,0 +1,85 @@
+package server
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path"
+)
+
+const defaultGatewayConfigPath = "~/.dfms/gateway_cfg.json"
```

Comment:
We need to separate gateway from dfms folder, cause it might work remotely on it's own machine. DFMS HTTP API allows us to do that. Let's change it to ".dfms_gateway".